vendor/github.com/docker/notary/README.md

```diff
@@ -78,18 +78,21 @@ to use `notary` with Docker images.
 
 ## Building Notary
 
-Prerequisites:
+Note that our [latest stable release](https://github.com/docker/notary/releases) is at the head of the
```

Comment:
s/Note that our/Notary's

Reply:
vendored code - we can address this in docker/notary